asapd-lite-installer/asapd-lite-installer-a4x-max-bm-cos.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: asapd-lite
+  namespace: kube-system
+  labels:
+    app: asapd-lite
```

Comment:
even though this label is useless, it feel awkard to have app here and k8s-app below. change to k8s-app here?